### PR TITLE
Saving photo

### PR DESCRIPTION
**Pull request by domanttas**
Added method to get Memory Stream from photo URI;
Note: previous method of getting Image from URI is redundant now, I was wrong, it did not return memory stream.
Added Unit Testing project;
Added unit test for GetStreamFromUri;

### CODE REVIEW

File: WhoIsThatUnitTests/ImageHandlerUnitTests.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+namespace WhoIsThatUnitTests
+{
+    public class ImageHandlerUnitTests
+    {
+        [Test]
+        public void GetStreamFromUri_ShouldReturn()
+        {
+            //Arrange
+            var uri = "https://whoisthatserverimages.blob.core.windows.net/images/Domantas_test.jpg";
```
Comment (tomasmuzas) [inline WhoIsThatUnitTests/ImageHandlerUnitTests.cs]: Hmm, most of the time, requests are mocked with a fake return value, and only response handling is tested (happy flow, exception handling etc).
You could say:
With any address I send GET to, return me X image or throw me Y exception (take a look at `FakeItEasy`).

Comment (tomasmuzas) [reply WhoIsThatUnitTests/ImageHandlerUnitTests.cs]: Anyway, nice to see that some tests are finally introduced :) dont forget to update them as your code updates

Comment (tomasmuzas) [reply WhoIsThatUnitTests/ImageHandlerUnitTests.cs]: Also, another thing. If you are testing class X, all of its depedencies are usually mocked, in order not to test same systems twice. Each dependency (ONLY if it is written by you) should have its separate unit test

Comment (domanttas) [reply WhoIsThatUnitTests/ImageHandlerUnitTests.cs]: Actually, I used FakeItEasy to mock database and DBSets while testing backend for this app. However, I couldn't come up with a solution on how to mock URI or some sort of WebClient, Http request. I'm aware that this test is crappy :D just couldn't figure it out

Comment (tomasmuzas) [reply WhoIsThatUnitTests/ImageHandlerUnitTests.cs]: It could look something like
`A.CallTo( () => fakeController.Get(A<string>._)).Returns(new Image())`

Comment (tomasmuzas) [reply WhoIsThatUnitTests/ImageHandlerUnitTests.cs]: Not sure if that would make much sense though, since your controllers dont have much logic in it

Comment (domanttas) [reply WhoIsThatUnitTests/ImageHandlerUnitTests.cs]: I'll try to figure something out, thanks for help!

File: WhoIsThatUnitTests/ImageHandlerUnitTests.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+using Shouldly;
+using WhoIsThat.Handlers;
+using Xamarin.Forms;
+
+namespace WhoIsThatUnitTests
+{
+    public class ImageHandlerUnitTests
+    {
+        [Test]
+        public void GetStreamFromUri_ShouldReturn()
```
Comment (tomasmuzas) [inline WhoIsThatUnitTests/ImageHandlerUnitTests.cs]: Just noticed, it probably should be like `ShouldReturnCorrectValue` or `ReturnsCorrectValue`?:D

**Comment by domanttas** [comment]
@tomasmuzas I think I successfully faked WebClient by writing wrappers and unit test should be okay right now, it would be cool if you could review it

**Pull request merged**